stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -892,7 +896,18 @@ private static void extractHttpsConfig(Annotation configInfo, Set<ListenerConfig
             if (trustStorePasswordAttrVal != null) {
                 listenerConfiguration.setTrustStorePass(trustStorePasswordAttrVal.getStringValue());
             }
-
+            if (certificateRevocationVerifierAttrValue.getBooleanValue() && cacheDelayAttrValue == null) {
+                throw new BallerinaException("cacheDelay value must be provided to enable certificate validation");
+            }
+            if (certificateRevocationVerifierAttrValue.getBooleanValue() && cacheSizeAttrValue == null) {
+                throw new BallerinaException("cacheSize value must be provided to enable certificate validation");
+            }
+            if (certificateRevocationVerifierAttrValue.getBooleanValue()) {
+                listenerConfiguration
```

Comment:
Shall we move above two validations inside this if condition? Then we can get rid of redundant certificateRevocationVer...AttrValue.getBooleanValue().